app/components/pages/Dashboard/operations.js

```diff
@@ -9,6 +9,7 @@ export const ALL_MANUSCRIPTS = gql`
         title
       }
       clientStatus
+      formState
```

Comment:
Maybe we can change the name to be something more explanatory, like `previousPage`, `lastVisitedPage` if that makes sense?

Reply:
`lastVisitedPage` :+1:

Reply:
This would require a migration, which should be automatically applied, but we haven't actually run a migration since we created Production and Staging last month so there may be some manual steps needed.

And in the spirit of getting small tickets/issues through, I suggest moving this change to a separate PR/issue.

Reply:
agree on move this to a separate ticket

Reply:
so new ticket raised #1097 for refactor the field name